shared-services/src/files/file-service.provider.ts

```diff
@@ -17,11 +18,15 @@ export class FileServiceProvider {
     return fileConfig
   }
 
-  private create(): void {
+  public static create(): void {
```

Comment:
This method shouldn't be necessary.  All `FileService`s should be created during `configure` (see above).

Reply:
Done.